resources/views/dashboard/widgets/previews/_members.blade.php

```diff
@@ -38,8 +38,8 @@
             @if (empty($member->character))
                 @continue
             @endif
-                <div class="font-extrabold">{{ $member->role }}</div>
-                <div>{!! $member->character->tooltipedLink() !!}</div>
+            <div class="font-extrabold" data-role="{{ Illuminate\Support\Str::slug($member->role) }}">{{ $member->role }}</div>
```

Comment:
Please make a new div that surrounds both divs, aka
```
<div class="grid grid-cols-2 gap-2" data-role="" data-status="">
   <div class="font-extrabold">...</div>
   <div>...</div>
</div>
```

So that our css wizards can stylise the whole row, not just fields individually :)